.github/dependabot.yaml

```
@@ -8,4 +8,9 @@ updates:
     go-dependencies:
       patterns:
       - "*"
-  open-pull-requests-limit: 5
\ No newline at end of file
+  open-pull-requests-limit: 5
+
+- package-ecosystem: github-actions
+  directory: /
+  schedule:
+    interval: daily
```

(Optional) if you prefer to group together Github/action related package updates, you can apply the following `groups` pattern to `github-actions`:


```suggestion
  - package-ecosystem: "github-actions"
    directory: "/"
    groups:
      github:
        patterns:
          - "actions/*"
          - "github/*"
    schedule:
      interval: "daily"
```